lib/tests/formulae.rb

```diff
@@ -216,6 +216,16 @@ def bottle_reinstall_formula(formula, new_formula, args:)
         test "brew", "install", @bottle_filename
       end
 
+      def all_deps_bottled?(formula)
```

Comment:
```suggestion
      def all_deps_bottled_no_older_versions?(formula)
```

or something? Could also consider just inlining into `build_bottle?`